docs/cip/README.md

```diff
@@ -461,247 +1988,671 @@ The Leios protocol may have to mitigate the following categories of threats.
+<!-- Simulation topology references -->
 
-| Persistent voters                                        | Certificate size                                                 |
-| -------------------------------------------------------- | ---------------------------------------------------------------- |
-| ![Fait-accompli voters](images/fait-accompli-voters.svg) | ![Fait-accompli certificate size](images/fait-accompli-cert.svg) |
+[mainnet-topology]:
+  https://github.com/input-output-hk/ouroboros-leios/blob/6d8619c53cc619a25b52eac184e7f1ff3c31b597/data/simulation/pseudo-mainnet/ReadMe.md
+  "Mainnet-like topology documentation"
+[pseudo-mainnet]:
+  https://github.com/input-output-hk/ouroboros-leios/blob/6d8619c53cc619a25b52eac184e7f1ff3c31b597/data/simulation/pseudo-mainnet/topology-v1.md
+  "Pseudo-mainnet topology"
+[mini-mainnet]:
+  https://github.com/input-output-hk/ouroboros-leios/blob/6d8619c53cc619a25b52eac184e7f1ff3c31b597/data/simulation/pseudo-mainnet/topology-v2.md
+  "Mini-mainnet topology"
+[topology-comparison]:
+  https://github.com/input-output-hk/ouroboros-leios/blob/6d8619c53cc619a25b52eac184e7f1ff3c31b597/analysis/sims/2025w30b/analysis.ipynb
+  "Topology comparison study"
+
+<!-- External resources -->
+
+[ripe-atlas]: https://atlas.ripe.net/ "RIPE Atlas"
+[cardano-foundation]: https://cardanofoundation.org/ "Cardano Foundation"
+
+<!-- License -->
+
+[apache-2.0]: http://www.apache.org/licenses/LICENSE-2.0 "Apache License 2.0"
+
+<!-- Footnotes -->
+
+[^fasort]: The Fait Accompli sortition scheme
+
+[^2]: Leios: Dynamic Availability for Blockchain Sharding (2025)
+
+[^leioscrypto]: Leios cryptography prototype implementation
+
+## Appendix
+
+<h3 id="appendix-a-requirements">Appendix A: Requirements for votes and certificates</h2>
+
+The voting and certificate scheme for Leios must satisfy the following
+requirements to ensure security, efficiency, and practical deployability:
+
+1. **Succinct registration of keys:** The registration of voting keys should not
+   involve excessive data transfer or coordination between parties. Ideally,
+   such registration would occur as part of the already existing operational
+   certificates and not unduly increase their size.
+
+2. **Key rotation:** The cryptographic keys used to sign Leios votes and
+   certificates _do not_ need to be rotated periodically because the constraints
+   on Leios voting rounds and the key rotation already present in Praos secure
+   the protocol against attacks such as replay and key compromise.
+
+3. **Deterministic signatures:** Deterministic signatures can guard against
```

Comment:
Deterministic signature are indeed less risky (for instance you do not need to worry about low entropy randomness) but are not strictly required. 
We only need deterministic random value for the lottery, and you are using VRFs for this matter.